src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsoffendercategorisationapi/model/entity/offendercategorisation/FormEntity.kt

```diff
@@ -0,0 +1,86 @@
+package uk.gov.justice.digital.hmpps.hmppsoffendercategorisationapi.model.entity.offendercategorisation
+
+import com.vladmihalcea.hibernate.type.json.JsonType
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.Table
+import org.hibernate.annotations.Type
+
+/**
+ *  assigned_user_id, user_id referred_by, approved_by,
+ *  assessed_by, security_reviewed_by cancelled_by all ** REACTED **
+ */
+@Entity
+@Table(name = "form", schema = "public")
+class FormEntity(
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  val id: Long = -1,
```

Comment:
I believe hibernate recognises a zero value as being transient rather than -1

Reply:
across moj repos there are many occurnaces declaring it as -1 but then again as 0, ill change to 0